packages/cf-component-box/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cf-component-box",
   "description": "Cloudflare Box Component",
-  "version": "1.4.3",
+  "version": "2.0.0",
```

Comment:
@koddsson you right, it looks suspicious cause publish appears in your branch commits. Please rebase from master to get latest packages versions.